apps/auth/main.py

```
@@ -0,0 +1,51 @@
+from fastapi import FastAPI, HTTPException, Response
+from fastapi import FastAPI
+import uvicorn
+import models
+from config import PASSWORD_MAX_LENGTH, PASSWORD_MIN_LENGTH, AUTH_PORT, ISSECURE
+import database as db
```

Ruff (0.8.2)

`database` imported but unused

Remove unused import: `database`

(F401)